web/src/components/CollapsibleList.js

```diff
@@ -0,0 +1,92 @@
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+
+class ListItem extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      expanded: props.initialExpanded
+    };
+  }
+
+  toggle = () => {
+    this.setState(prev => ({ expanded: !prev.expanded }));
+  };
+
+  render() {
+    const { children, deleteItem, header } = this.props;
+
+    const childClass = this.state.expanded ? '' : 'd-none';
+
+    return (
+      <Fragment>
+        <div className="mb1 h5 flex justify-between">
+          <button
+            type="button"
+            className={`btn btn-no-focus p1 flex-auto left-align bg-blue-light ${
+              deleteItem ? 'rounded-left' : 'rounded'
+            }`}
+            onClick={this.toggle}
```

Comment:
I feel like there should probably be ARIA attributes (`aria-controls` on the button and a corresponding `id` on the collapsible panel) on this somehow. But the structure of this may be changing in #1077, so it's probably not worth stressing over too much yet.

Reply:
Good point.  Do you think we should make an issue just so we remember that the `aria-controls` is missing?